### ISSUE TITLE
v1.1.18 checksum mismatch

### ISSUE THREAD

**Issue by dajohi** [miekg/dns#1014]
I am getting the following when I put v1.1.18 in my go.mod:

```
$ env GOPROXY=direct GO111MODULE=on go mod tidy
go: downloading github.com/miekg/dns v1.1.18
verifying github.com/miekg/dns@v1.1.18: checksum mismatch
	downloaded: h1:RP3QrPVgq8fLiw0mjYwVGNaIsCc9MxNFlRHu8PlAeiY=
	sum.golang.org: h1:S82KA03bsvMvziY41d0WitiplMCt8QhawbSQLtoqsdI=

SECURITY ERROR
This download does NOT match the one reported by the checksum server.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

For more information, see 'go help module-auth'.
```

**Comment by ThreeFx** [comment]
I get the same error.

**Comment by miekg** [comment]
The shitshow that is go modules keeps on giving. I'll push a new version
shortly (or stop doing version releases altogether because nothing improved
with that)

**Comment by jrick** [comment]
did you force push the tag?

**Comment by miekg** [comment]
Yes, and deleted it. Tagged my local repo that didn't have all upstream
changes yet

**Comment by jrick** [comment]
Ok, well that explains it.  Unfortunately if the version has already hit the proxy and sum servers, that is indistinguishable from a MITM attack.

The proper method is to push a v1.1.18+deprecated tag to indicate it should not be used, and then create a v1.1.19 with the desired fixes.

**Comment by miekg** [comment]
I'm going to release a new tag. Not jumping through go modules bull crap.

**Comment by miekg** [comment]
released 1.1.19
